File: Itan.Core/LandingPageNews.cs
using System;

namespace Itan.Core
{
    public class LandingPageNews
    {
        public string Author { get; set; }
        public Guid ChannelId { get; set; }
        public string Title { get; set; }
        public Guid Id { get; set; }
        public DateTime Published { get; set; }
        public string Link { get; set; }
        public string Image { get; set; }
        public string Content { get; set; }
        public string ContentLink { get; set; }
    }
}

File: Itan.Application.UI/src/app/components/landing-page/landing-page.news.ts
import {LandingPageNewsViewModel} from "../../../server/Itan/Core/LandingPageNewsViewModel";
import {NewsContent} from "./news.content";

export class LandingPageNews {
    constructor(vm: LandingPageNewsViewModel) {
        this.content = new NewsContent();
        this.viewModel = vm;
    }

    viewModel: LandingPageNewsViewModel;
    content: NewsContent;
}
